tests/test_polygons.py

```diff
@@ -13,6 +22,42 @@ def test_to_from_geojson_file(data_path):
         assert polygons2 == polygons
 
 
+def test_geometry_simplify(data_path):
+    # load data
+    print('loading')
```

Comment:
Misc print statements for local testing but not crucial. Maybe there is already another approach for logging inside tests, or better to avoid overall?